examples_test.go

```diff
@@ -5,7 +5,7 @@ import (
 	"runtime"
 	"time"
 
-	"gopkg.in/alexcesaro/statsd.v2"
+	"github.com/joeycumines/statsd"
```

Comment:
why change the path

Reply:
that was an ask from @joeycumines in the last PR if we wanted to use go modules. I'm moving this change to #3